lib/noop.js

```diff
@@ -0,0 +1 @@
+module.exports = require('lodash/utility/noop')
```

Comment:
`module.exports = function noop (){};` is like 1000% times saner to user right here